molecule/ca-renew/prepare.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Prepare
+  hosts: all
+  vars:
+    ca_ca_host: ca_default
+    ca_logstash: true
+    ca_etcd: true
+    ca_etcd_group: molecule
+    ca_ca_days: 5
+    ca_valid_time: "+14d"
+    ca_renew: true
+  tasks:
+
+    - name: Install Python libraries
+      pip:
+        name: cryptography>= 1.2.3
+
+    - name: Install packages for RHEL
+      package:
+        name:
+          - iproute
+          - NetworkManager
+      when: ansible_os_family == "RedHat"
+
+    - name: Start NetworkManager
+      service:
+        name: NetworkManager
+        state: started
+        enabled: yes
+      when: ansible_os_family == "RedHat"
+
+    - name: Gather facts again to define ansible_default_ipv4
+      setup:
+
+    - name: "Include CA role"
+      include_role:
+        name: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') | basename }}"
```

Comment:
Do you need to run the role twice for the renewal process?

Reply:
Yes, I do need to run it twice to emulate the renewing process.

The first time I run the role it creates "old" certificates with an expiration date of roundabout 1 week.
The second time in converge.yml it'll renew the certificates and that's what we need to test.

Reply:
Thanks for the explanation!